unity-environment/Assets/ML-Agents/Editor/MLAgentsEditModeTest.cs

```diff
@@ -240,31 +242,42 @@ public void TestAgent()
 
             AgentEnableMethod.Invoke(agent1, new object[] { aca });
             AcademyInitializeMethod.Invoke(aca, new object[] { });
-            AgentEnableMethod.Invoke(agent2, new object[] { aca });
 
             MethodInfo AcademyStepMethod = typeof(Academy).GetMethod("_AcademyStep",
                            BindingFlags.Instance | BindingFlags.NonPublic);
 
+            int numberAgent1Reset = 0; 
```

Comment:
ensure that you convert tabs to spaces, indentation starting from lines 243 seems off.

Reply:
Tabs converted